### PR TITLE
Commit the Stream Processor vagrant version and configuration changes

### LINKED ISSUE #24: Virtualbox packager includes stray information
**Description:**
Virtualbox packager includes stray information unless the following config is used inside Vagrantfile
```
  myvm.vm.provider "virtualbox" do |vb|
      # workaround: if the following line is not present, you get the bug
      vb.customize [ 'modifyvm', :id, '--uartmode1', 'disconnected']
  end
```
Vagrant box packaging (`vagrant package --base ... --output ...`) will result in the inclusion of a hardcoded (and apparently totally unrelated) user file path (usually related to the user creating the box, eg` /home/user/<filename>`) that can be found in the `box.ovf` (XML) file, resulting in errors when another user tries to install (`vagrant up`) that box.

This .ovf file is probably used to configure serial ports, but in this case, a local log file path somehow ends up in this setting.

(originally submitted at https://www.virtualbox.org/ticket/17513)

**Suggested Labels:**
Type/Bug
Severity/High

**Suggested Assignees:**
@VimukthiPerera 

**Affected Product Version:**
All

**Steps to reproduce:**

```
vagrant --updates up
```

Error message:
```
==> wso2-mysql: Booting VM...
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr are shown below.

Command: ["startvm", "74cd8903-d6b6-455b-9854-1d5b58c136f3", "--type", "headless"]

Stderr: VBoxManage: error: RawFile#0 failed to create the raw output file /home/vimukthi/Desktop/git-repo/vagrant-boxes/vagrant-boxes/ubuntu-bionic-18.04-cloudimg-console.log (VERR_FILE_NOT_FOUND)
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component ConsoleWrap, interface IConsole
```

**Related Issues:**
https://github.com/hashicorp/vagrant/issues/9425

### PR DESCRIPTION
**Pull request by chamindi-a**
## Purpose
Commit the Stream Processor vagrant version and configuration changes for the version 4.4.0
resolves #24

## Goals
Version and configuration changes performed for vagrant-sp files

## Test environment
vagrant version: 2.2.4
OS Version: macOS 10.136
Vbox: 5.2.14

### CODE REVIEW

File: worker/repository/conf/deployment.yaml [hunk 1]
```diff
@@ -129,16 +106,15 @@ databridge.config:
         sslPort: '9711'
         tcpReceiverThreadPoolSize: '100'
         sslReceiverThreadPoolSize: '100'
-        hostName: 172.28.128.4
-
+        hostName: 0.0.0.0
+        host: 172.28.128.4
```
Comment (VimukthiPerera) [inline worker/repository/conf/deployment.yaml]: @chamindi-a Please recheck this. Are there two parameters? `host` and `hostName`?

File: .metadata [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        new_value: "{$arg.new_version}"#
+#README.md changes
+  - file_path: "README.md"
+    relative_path: ~
+    file_type: "txt"
+    configurations:
+      - operation: "replace"
+        current_value: "v4.3.0.1 "
+        new_value: "v4.4.0.1 "
+
```
Comment (VimukthiPerera) [inline .metadata]: Shall we remove the extra line at the EOF?

File: .metadata [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+# ----------------------------------------------------------------------
+---
+# command line arguments
+params:
+  existing_version: 4.3.0
+  new_version: 4.4.0
+  wso2sp: "wso2sp-4.4.0"
+# files based configurations
+files:
+#Worker Configuration and Version changes
```
Comment (VimukthiPerera) [inline .metadata]: Let's keep consistency in the comments. Start with a # and space and then start the comment with a simple letter.
Eg: `# sample comment`

File: .metadata [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+      - operation: "replace"
+        current_value: "{$arg.existing_version}"
+        new_value: "{$arg.new_version}"#
+# readme.md changes
+  - file_path: "README.md"
+    relative_path: ~
+    file_type: "txt"
+    configurations:
+      - operation: "replace"
+        current_value: "v4.3.0.1 "
```
Comment (jsdjayanga) [inline .metadata]: Can you parameterize this?

**Review by VimukthiPerera — APPROVED** [review]

**Pull request merged**